src/components/ImageGallery/components/ImageGalleryFooter.tsx

```diff
@@ -25,7 +25,9 @@ import type { Photo } from '../ImageGallery';
 
 import type { DefaultUserType, UnknownType } from '../../../types/types';
```

Comment:
do we still need this change without the styleguidist renders?

Reply:
Yeah, still seemed to be breaking without it!! I didn't get much time to look too much into it